#!/usr/bin/env python3
# -*- coding: utf-8 -*-
# (c) ACE 

import os

class Config(object):
    # get a token from @BotFather
    pass
    """
    BOT_TOKEN = os.environ.get("BOT_TOKEN", "7123545054:AAE2GLTcdZoWtop8d30L3jMNMKDhAJh6wsM")
    API_ID = int(os.environ["API_ID", 23537514]
    API_HASH = os.environ["API_HASH", "c5ec016af7d2cb637278b715f4f5a389"]
    AUTH_USERS = "7134852010"""
# jisko bhi is repo ko deploy karne me error aa raha ho ( https://t.me/Stubborn1223 ) ye mere TG ka id ha message karke iska screen shot send kar dena me error solve kar dunga. 
